MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+3.  Application to fetch kubernetes objects metadata from a kubernetes cluster (deployed as a deployment)
+
+All the applications in SCK use Fluentd to work with logs, metrics and objects. Fluentd has significant performance issues when used to fetch logs in a Kubernetes cluster with pods that have very high throughput.  
+
+Splunk OpenTelemetry Collector for Kubernetes provides significant performance improvements over the SCK through use of the OpenTelemetry Collector agent and native OpenTelemetry functionality for logs collection rather than Fluentd. You can learn more about the performance characteristics of this new application [here](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#performance-of-native-opentelemetry-logs-collection).
+
+Splunk OpenTelemetry Collector for Kubernetes has 2 components/applications:
+1.  Application to fetch logs and traces from a kubernetes cluster (deployed as a DaemonSet)
+2.  Application to fetch metrics and objects from a kubernetes cluster (deployed as a DaemonSet).
+
+No application currently exists for fetching kubernetes objects metadata from a kubernetes cluster. 
+ 
+### Changes in Logs, Metrics and Objects
+
+| Logging | Metrics | Objects |
+|---|---|---|
+| Redhat UBI docker images for our applications are no longer available, as we now use scratch images | The naming convention of the metrics used in SCK-OTEL has changed and there are some minor differences in the names of the metrics | Not implemented/Not available |
+| SCK-OTEL does not support AWS Firelens. AWS Firelens uses Fluentd and we can achieve similar outcomes with opentelemetry. | Changes in the number of metrics (fewer metrics available) and additional metrics for the open telemetry collector |  |
+|  |  |  |
+
+
+### Changes in Logs in Splunk OpenTelemetry Connector for Kubernetes(SCK-OTEL)
+* Redhat UBI docker images for our applications are no longer available, as we now use scratch images.
+* Splunk OpenTelemetry Collector for Kubernetes does not support AWS Firelens. AWS Firelens uses Fluentd and we can achieve similar outcomes with OpenTelemetry.
+
+### Changes in Metrics in Splunk OpenTelemetry Connector for Kubernetes(SCK-OTEL)
+* The naming convention of the metrics used in Splunk OpenTelemetry Collector for Kubernetes follows the OpenTelemetry specification and is different than SCK. You will observe some minor differences in the names of the metrics. 
+* Previously in SCK, you could get a large number of metrics from various APIs for kubernetes. However, in recent versions of kubernetes 1.18+, these API sources are disabled by default and there are fewer metrics available. The previous list of metrics can be found [here](https://github.com/splunk/fluent-plugin-kubernetes-metrics/blob/master/metrics-information.md).
+
+Some additional metrics that are available in Splunk OpenTelemetry Collector for Kubernetes are metrics about the OpenTelemetry Collector itself.
```

Comment:
What's the relevancy of this statement? Is there a list or link to these metrics?

Reply:
yea, it is the metrics that starts with `otelcol_` from the list below